pkg/command/recommendation/options.go

```diff
@@ -170,6 +175,60 @@ func (opts *ContainerResourcesOptions) Apply(configuration *[]applications.Confi
 	}
 }
 
+// HPAResourcesOptions contains options for building the recommender configuration
+// for optimizing hpa resources.
+type HPAResourcesOptions struct {
+	TargetUtilizationMax map[string]int
+	TargetUtilizationMin map[string]int
+}
+
+func (opts *HPAResourcesOptions) AddFlags(cmd *cobra.Command) {
```

Comment:
This needs to be called from [here](https://github.com/thestormforge/optimize-go/blob/efd315f9fbf63ff6eb3865d50d34741e425b7a48/pkg/command/applications.go#L176).